deploy/containers/credhub_setup/credhub_info.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+// credhub_info.go contains functions to determine the credhub information
```

Comment:
Same thing here about structure. If it's doing `credhub` stuff, let's put it on a `credhub` package!